.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           name: Build
           command: |
             python3 -m venv venv/
-            venv/bin/pip install pip-tools --constraint requirements.in
+            venv/bin/pip install --no-deps -r requirements.txt
             venv/bin/pip-sync --pip-args=--no-deps
```

Comment:
constraints cannot contain extras, and requirements.in now contains `google-cloud-bigquery-storage[fastavro]`, so instead of installing just pip-tools this now installs everything.

`pip-sync` is still necessary because it uninstalls things that have been removed.